specification/monitor/resource-manager/Microsoft.Insights/stable/2018-09-01/baseline_API.json

```diff
@@ -98,6 +98,68 @@
             "Get Metric for metadata": { "$ref": "./examples/GetBaselineMetadata.json" }
           }
         }
+      },
+      "/{resourceUri}/providers/microsoft.insights/baseline": {
+        "get": {
+          "tags": [
+            "Baseline"
+          ],
+          "operationId": "Baseline_Get",
+          "description": "**Gets the baseline values for a resource**.",
+          "parameters": [
+            {
+              "$ref": "#/parameters/ExtendedResourceUriParameter"
+            },
+            {
+              "$ref": "#/parameters/MetricNamesParameter"
+            },
+            {
+              "$ref": "#/parameters/TimespanParameter"
+            },
+            {
+              "$ref": "#/parameters/IntervalParameter"
+            },
+            {
+              "$ref": "#/parameters/AggregationParameter"
+            },
+            {
+              "$ref": "#/parameters/SensitivitiesParameter"
+            },
+            {
+              "$ref": "#/parameters/BaselineResultTypeParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "#/parameters/MetricNamespaceParameter"
+            },
+            {
+              "name": "$filter",
+              "in": "query",
+              "type": "string",
+              "description": "The **$filter** is used to describe a set of dimensions with their concrete values which produce a specific metric�s time series, in which a baseline is requested for.",
```

Comment:
Please, fix the � symbol. The file should use UTF-8 encoding witout BOM.

Reply:
I fixed the bad symbol.